install-scripts/skm-install.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
+
+# If the kernel is a version of macOS, run the mac-specified installer.
+if [[ `uname` = Darwin ]]; then
+    ./mac/skm-install-mac.sh
```

Comment:
**question:** Would this relative path work when the user runs the following command as per the installation [doc](https://splashkit.io/articles/installation/mac/step-2/)? 🤔 

```sh
bash <(curl -s https://raw.githubusercontent.com/splashkit/skm/master/install-scripts/skm-install.sh)
```

I guess locally, we can test with the following command

```sh
bash <(./install-scripts/skm-install.sh)

Reply:
Fair point. Neither will work since `curl` is only going to grab the one file. Short of hard-coding links to GitHub installation scripts in the installer (which may be a circular dependency), either the installation doc will need to be updated to have a macOS specific installation link, or the team will need to merge the separate installers into one big conditional file (once the others are written). I think I'd rather the single file to avoid complexity. I will remove this for now and we can address at a later date.

Reply:
> Neither will work since curl is only going to grab the one file. 

Yep, I suspect that too. Thank you for confirming. I agree with the proposed approach.

> either the installation doc will need to be updated to have a macOS specific installation link, or the team will need to merge the separate installers into one big conditional file (once the others are written).

Could you please raise a card so we don't forget this? 🙏🏼